packages/commonwealth/client/scripts/views/components/TokenFinder/useTokenFinder.ts

```diff
@@ -20,7 +22,17 @@ const useTokenFinder = ({
       nodeEthChainId,
     });
 
+  const nativeTokenMetadata: GetTokenMetadataResponse = {
+    decimals: 18,
+    logo: '',
+    name: 'ETH', // TODO: get native eth name/symbol
```

Comment:
Are you gonna work on this TODO in this PR? If not, please create ticket and link it here in the comment.

Reply:
https://github.com/hicommonwealth/commonwealth/issues/10108